libethereum/EthereumHost.h

```diff
@@ -51,20 +51,95 @@ class TransactionQueue;
 class BlockQueue;
 class BlockChainSync;
 
+struct EthereumPeerStatus
+{
+    /// What, if anything, we last asked the other peer for.
+    Asking m_asking = Asking::Nothing;
+    /// When we asked for it. Allows a time out.
+    std::atomic<time_t> m_lastAsk;
+    /// Peer's protocol version.
+    unsigned m_protocolVersion;
+    /// Peer's network id.
+    u256 m_networkId;
```

Comment:
What is this?

Reply:
Network ID that the peer sends us in eth Status message